R/languages.R

```diff
@@ -3,6 +3,7 @@
 #' Lists the languages available for MDT implementations.
 #' @export
 mdt_languages <- function() {
+  c("EN", "DE", "DE_F", "FR", "RU")
```

Comment:
Are you sure this change is right? I don't see Latvian listed, or Spanish, or Italian